### ISSUE TITLE
need more help about the tortoise-diffusion model training

### ISSUE THREAD

**Issue by JohnHerry** [neonbjb/DL-Art-School#21]
I am not familiar with the ddpm,  but when try to train the ddpm in of tortoise-tts in this project, there are some different loss  components, "loss_ddpm_total", "loss_diffusion_loss", "loss_mel_surrogate", "loss_upper_mse", "loss_var_loss",   can anyone help to explain what are those components do? and how can I judge the diffusion model converage with any of the above components?  and , If we do fine-tune training with AR model output latent, is there any change about the loss usage above?
